src/env.h

```diff
@@ -562,6 +565,7 @@ class Environment {
   int handle_cleanup_waiting_;
 
   uint32_t* heap_statistics_buffer_ = nullptr;
+  uint32_t* heap_space_statistics_buffer_ = nullptr;
```

Comment:
We could in theory share the memory between these two.  Not that important though.